Changes

```diff
@@ -2,6 +2,9 @@ Revision history for Perl module WWW::YouTube::Download
 
 {{$NEXT}}
 
+0.64	   2020-08-27 16:00:00Z
+    - Get youtube-playlists working again. (GH#40) (isync)
```

Comment:
I think this should be 60 rather than 40?

Reply:
Hi Olaf,
I've updated the pull request. For example, this line, well.. it depends on how you look at it. I though I fixed issue 40 and that's what I notes in changes, now I've clarified it